libs/cases/frontend/view/src/lib/components/case.mocks/const.ts

```diff
@@ -63,32 +92,178 @@ export const cases: ICaseMock[] = [
+		//Push Card
+		//id: 'AA-000',
+		facilityId: 1,
+		icon: 'battery-empty',
+		notification: '99+',
+		//heading: 'Fix the Aquarium Pump',
+		//subheading: 'The pump in the aquarium is not working properly',
+		//variant: 'warning',
+
+		//Case
+		handle: 'AA-005',
+		dueDate: '2023-08-10T03:52:14.301Z',
+		notifyAssignee: false,
+		title: 'test WOM 1',
+		type: 'warning',
+		status: 'CANCELLED',
+		assignedTo: 'Mike Jefferson von Schmitz man',
+		description:
+			'this case is for changing and cleaning the floppy disks for the nuclear launch codes',
+		source: 'AHM',
+		priority: '0',
+		createdBy: 'test@test.com',
+		createdDate: '2023-08-10T03:52:14.301Z',
+		eTag: '1702540787672',
+		modifiedBy: 'test@test.com',
+		modifiedDate: '2023-08-10T03:52:14.301Z',
+		overdue: false,
+	},
+	{
+		//Push Card
+		//id: 'AA-000',
+		facilityId: 1,
+		icon: 'battery-empty',
+		notification: '99+',
+		//heading: 'Fix the Aquarium Pump',
+		//subheading: 'The pump in the aquarium is not working properly',
+		//variant: 'warning',
+
+		//Case
+		handle: 'AA-006',
+		dueDate: '2023-08-10T03:52:14.301Z',
+		notifyAssignee: false,
+		title: 'test WOM 1',
+		type: 'warning',
+		status: 'ARCHIVED',
+		assignedTo: 'Mike Jefferson von Schmitz man',
+		description:
+			'this case is for changing and cleaning the floppy disks for the nuclear launch codes',
+		source: 'AHM',
+		priority: '0',
+		createdBy: 'test@test.com',
+		createdDate: '2023-08-10T03:52:14.301Z',
+		eTag: '1702540787672',
+		modifiedBy: 'test@test.com',
+		modifiedDate: '2023-08-10T03:52:14.301Z',
+		overdue: false,
+	},
+	{
+		//Push Card
+		//id: 'AA-000',
+		facilityId: 1,
+		icon: 'battery-empty',
+		notification: '99+',
+		//heading: 'Fix the Aquarium Pump',
+		//subheading: 'The pump in the aquarium is not working properly',
+		//variant: 'warning',
 
-        //Case
-        handle: 'AA-099',
-        dueDate: '2024-09-10T03:52:14.301Z',
-        notifyAssignee: false,
-        title: 'Is this overdue?',
-        type: 'warning',
-        status: '0',
-        assignedTo: 'Dan the man',
-        description: 'Dan has to stop the water pipe from leaking',
-        source: 'AHM',
-        priority: '0',
-        createdBy: 'why@care.su',
-        createdDate: '2023-08-10T03:52:14.301Z',
-        eTag: '1702540787672',
-        modifiedBy: 'test@test.com',
-        modifiedDate: '2023-08-10T03:52:14.301Z',
-        overdue: true,
-    },
+		//Case
+		handle: 'AA-008',
+		dueDate: '2023-08-10T03:52:14.301Z',
+		notifyAssignee: false,
+		title: 'test WOM 1',
+		type: 'warning',
+		status: 'OPEN',
+		assignedTo: 'Mike Jefferson von Schmitz man',
+		description:
+			'this case is for changing and cleaning the floppy disks for the nuclear launch codes',
+		source: 'AHM',
+		priority: 'EMERGENCY',
+		createdBy: 'test@test.com',
+		createdDate: '2023-08-10T03:52:14.301Z',
+		eTag: '1702540787672',
+		modifiedBy: 'test@test.com',
+		modifiedDate: '2023-08-10T03:52:14.301Z',
+		overdue: false,
+	},
 ];
```

Comment:
why do u use a const here and not fake the data with like faker-js?

Reply:
Up to here I have not implemented faker-js. I can take a look at it this evening, but I doubt that it is a priority since it is a question of time until we can get "real" data.

Reply:
I tried it. It can be done but it the breaks the detail view. It is not worth the effort here.

Reply:
This is what chatgpt would do



```ts
import { faker } from '@faker-js/faker';

function generateCase() {
  return {
    handle: `AA-${faker.datatype.number({ min: 1, max: 999 }).toString().padStart(3, '0')}`,
    dueDate: faker.date.future().toISOString(),
    notifyAssignee: faker.datatype.boolean(),
    title: faker.lorem.words(3),
    type: 'warning',
    status: faker.helpers.arrayElement(['CANCELLED', 'ARCHIVED', 'OPEN']),
    assignedTo: faker.name.fullName(),
    description: faker.lorem.sentence(),
    source: 'AHM',
    priority: faker.helpers.arrayElement(['0', 'EMERGENCY']),
    createdBy: faker.internet.email(),
    createdDate: faker.date.recent().toISOString(),
    eTag: faker.datatype.uuid(),
    modifiedBy: faker.internet.email(),
    modifiedDate: faker.date.recent().toISOString(),
    overdue: faker.datatype.boolean(),
  };
}

function generatePushCard() {
  return {
    facilityId: 1,
    icon: 'battery-empty',
    notification: '99+',
    ...generateCase(),
  };
}

const cases = Array.from({ length: 5 }, generatePushCard);

console.log(JSON.stringify(cases, null, 2));

```

Reply:
Yeah I had something similar to trouble was, that when switching to create view it would simply show the deatil page without any details, not even empty boxes. I assume it generates when ever you switch sites it regenerates the cases.

Reply:
I got it to work is part of the last commit.

Reply:
Super!

Reply:
The only thing i would now suggest is deleting all the unused code :) 
Get rid of all the chunks of data code.